### PR TITLE
SW-2750 Add converted value to quantity box on accession page

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2023-01-25 at 17 32 09
Screenshot 2023-01-25 at 17 32 14

### CODE REVIEW

**Comment by karthikbtf** [comment]
@constanzauanini there are 0 files changed in this PR 🤣

**Comment by constanzauanini** [comment]
> @constanzauanini there are 0 files changed in this PR 🤣

Oh, no! It's because I committed in the wrong branch and then reverted the changes there. I will fix it

**Comment by constanzauanini** [comment]
> @constanzauanini there are 0 files changed in this PR 🤣

Updated!

File: src/units.ts [hunk 1]
```diff
@@ -53,3 +53,46 @@ export type InitializedUnits = {
...
+      return `${(value * 2.20462).toPrecision(2)} Pounds`;
+    }
+    case 'Milligrams': {
+      return `${(value * 0.000035274).toPrecision(2)} Ounces`;
+    }
+    case 'Pounds': {
+      return `${(value * 0.453592).toPrecision(2)} Kilograms`;
+    }
+    case 'Ounces': {
+      return `${(value * 28.3495).toPrecision(2)} Grams`;
```
Comment (karthikbtf) [inline src/units.ts:86]: need to localize the units here..

**Review by karthikbtf — APPROVED** [review]
one localization fix required

**Review by karthikbtf — APPROVED** [review]
Looks good, thanks for the localization fix.

**Pull request merged**